app/controllers/events/view/tickets/attendees/list.js

```diff
@@ -34,5 +34,17 @@ export default Controller.extend({
       template       : 'components/ui-table/cell/events/view/tickets/attendees/cell-action',
       disableSorting : true
     }
-  ]
+  ],
+  actions: {
+    toggleCheck(attendee) {
```

Comment:
Rename this action as `toggleCheckIn`, rest looks good.

Reply:
@CosmicCoder96 alright, will do

Reply:
@CosmicCoder96 alright, will do

Reply:
Request acknowledged, kindly review @CosmicCoder96